content/en/metrics/guide/dynamic_quotas.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Dynamic Quotas for Metrics
+further_reading:
+- link: "/account_management/billing/custom_metrics/?tab=countrate"
+  tag: "Documentation"
+  text: "Custom Metrics Billing"
+- link: "/metrics/guide/custom_metrics_governance/"
+  tag: "Guide"
+  text: "Best Practice for Custom Metric Governance"
+- link: "https://www.datadoghq.com/blog/metrics-without-limits/"
+  tag: "Blog"
+  text: "Dynamically control your custom metrics volume with Metrics without Limits™"
+algolia:
+  tags: ['custom metrics']
+is_beta: true
+private: true
+---
+
+## Overview
+
+{{< callout url="#" btn_hidden="true">}}
```

Comment:
Heh same comment for this beta banner - we're rebranding all the Betas to Preview so can the language match here?

Reply:
@estherk15 Does the is_beta variable need to change to reflect Preview?

Reply:
@ijkaylin thanks for flagging that, updated in [85f99a9](https://github.com/DataDog/documentation/pull/25262/commits/85f99a92217e17cbb07208f26c0e39338a3dc356). 
@ouracheal the `is_beta` variable can be left as it is for now; this doesn't generate text on the page itself.

Reply:
Thanks so much @buraizu !